src/tests/unit/tests/electron/views/screenshot/screenshot-view.test.tsx

```diff
@@ -1,19 +1,43 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
 import { ScreenshotView } from 'electron/views/screenshot/screenshot-view';
 import { ScreenshotViewModel } from 'electron/views/screenshot/screenshot-view-model';
 import { shallow } from 'enzyme';
 import * as React from 'react';
 
 describe('ScreenshotView', () => {
-    it('renders', () => {
```

Comment:
I think we want to keep a test of the happy path where it renders the available screenshot layout